src/resources/filters/quarto-pre/table-classes.lua

```diff
@@ -0,0 +1,88 @@
+-- table-classes.lua
+-- Copyright (C) 2020-2023 Posit Software, PBC
+
+-- helper for checking whether a single value is present in a table
+function contains(list, x)
+	for _, v in pairs(list) do
+		if v == x then return true end
+	end
+	return false
+end
+
+-- propagate table classes to the table's <table> element
+function tableTableTag() 
+  return {
+    Table = function(el)
+      if tcontains(el.attr.classes) then
```

Comment:
Are we missing a parameter here?

Relatedly, if you're using `tcontains` here, why do we define `contains` just before that?